lib/components/HexOnlyEditableField.jsx

```diff
@@ -230,26 +235,19 @@ class HexOnlyEditableField extends React.Component {
         for (let i = 0; i < valueArray.length; i += 1) {
             const value = valueArray[i];
             if (value.length % 2 !== 0) {
-                return { valid: false, validationMessage: 'Please enter full bytes (pairs of hexadecimals)' };
+                return {
+                    valid: false,
+                    validationMessage:
+                        'Please enter full bytes (pairs of hexadecimals)',
+                };
             }
         }
 
         return { valid: true, validationMessage: 'Valid value' };
     }
 
     render() {
-        const {
-            showText,
-            value,
-            keyPressValidation,
-            completeValidation,
-            formatInput,
-            onBeforeBackspace,
-            onBeforeDelete,
-            onChange,
-            title,
-            ...props
-        } = this.props; // pass along all props except these
+        const { showText, value, title, ...props } = this.props; // pass along all props except these
```

Comment:
Noticing that several props have disappeared here, assume it's deliberate?

Reply:
Yeah they were not used.